### PR TITLE
release: Mention the container images used to build the project

### PR DESCRIPTION
**Pull request by fidencio**
This is a small step towards build reproducibility.

Fixes: #7412

### CODE REVIEW

File: tools/packaging/release/release-notes.sh [hunk 1]
```diff
@@ -135,6 +135,25 @@ ${image_info}
...
+build reproducibility we publish those container images, and when those are used combined with the version
+of the projects listed as part of the "versions.yaml" file, users can get as close to the environment we
+used to build the release artefacts.
+* Kernel (on all its different flavours): $(get_kernel_image_name)
+* OVMF (on all its diferent flavours): $(get_ovmf_image_name)
+* QEMU (on all its different flavurs): $(get_qemu_image_name)
+* shim-v2: $(get_shim_v2_image_name)
+* virtiofsd: $(get_virtiofsd_image_name)
+
+The users who want to rebuild the tarballs using exactly the same images can simly use the following environment
```
Comment (dborquez) [inline tools/packaging/release/release-notes.sh:149]:
```suggestion
The users who want to rebuild the tarballs using exactly the same images can simply use the following environment
```

**Review by dborquez — CHANGES_REQUESTED** [review]

File: tools/packaging/release/release-notes.sh (hunk 1 above)
Comment (fidencio) [reply tools/packaging/release/release-notes.sh]: Fixed, thanks!

**Review by GabyCT — APPROVED** [review]

**Review by dborquez — APPROVED** [review]

**Pull request merged**